### PR TITLE
user_saml tests sometimes fail because the connection ain't ready

### PR DESCRIPTION
**Pull request by blizzz**
Signed-off-by: Arthur Schiwon <blizzz@arthur-schiwon.de>

### CODE REVIEW

**Comment by blizzz** [comment]
Noooo the docker build fails :(

**Comment by skjnldsv** [comment]
In other tests (like viewer with cypress) we use the wait-on binary to wait for an address to be ready ;)

**Comment by blizzz** [comment]
@skjnldsv the what? :) u got a pointer for me?

**Comment by skjnldsv** [comment]
> @skjnldsv the what? :) u got a pointer for me?

Of course :)
https://github.com/nextcloud/viewer/blob/6e0ee6404f82ddf306db4359db9d77c6688e00ff/.drone.yml#L57

**Comment by blizzz** [comment]
this makes more sense of course … let me try with netcat :)

File: user_saml_shibboleth-php7.2/start.sh [hunk 1]
```diff
@@ -17,4 +17,16 @@ sed -i "s/^-Xmx.*$/-Xmx$JETTY_MAX_HEAP/g" /opt/shib-jetty-base/start.ini
 
 apachectl &
 /usr/sbin/ns-slapd -D /etc/dirsrv/slapd-dir &
-sleep 4 && /etc/init.d/jetty run &
+
+# wait for LDAP
+for i in 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30
```
Comment (skjnldsv) [inline user_saml_shibboleth-php7.2/start.sh]: :grin:

Comment (blizzz) [reply user_saml_shibboleth-php7.2/start.sh]: POSIX compat

**Comment by blizzz** [comment]
Seems to work \o/

**Pull request merged**